app/controllers/api/stages_controller.rb

```diff
@@ -18,10 +16,19 @@ def clone
     end
   end
 
+  def duplicable
+    render json: @project.duplicable_stage
+  end
+
+  def put_duplicable
```

Comment:
chose_duplicable / mark_duplicable ?

On Fri, Aug 12, 2016 at 3:52 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/controllers/api/stages_controller.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r74671414>:
>
> > @@ -18,10 +16,19 @@ def clone
> >      end
> >    end
> >
> > +  def duplicable
> > +    render json: @project.duplicable_stage
> > +  end
> > +
> > +  def put_duplicable
>
> Sorry. #put_duplicable is to change which stage is marked duplicable.
> Only one stage per project is "duplicable". The #duplicable method is
> simply the currently marked stage on the project that is duplicable. Clone
> is just clone.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/122035a3f8635be3fd046f6620704db40db5d7c0#r74671414>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ2FRzyjFGV0ZsJblBRgHoD0LK-XAks5qfPkxgaJpZM4Jhtog>
> .
>